CHANGELOG.md

```diff
@@ -790,6 +790,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Deprecated `PrecisionPlugin.master_params()` in favor of `PrecisionPlugin.main_params()` ([#10105](https://github.com/PyTorchLightning/pytorch-lightning/pull/10105))
 - Deprecated `lr_sch_names` from `LearningRateMonitor` ([#10066](https://github.com/PyTorchLightning/pytorch-lightning/pull/10066))
 - Deprecated `ProgressBar` callback in favor of `TQDMProgressBar` ([#10134](https://github.com/PyTorchLightning/pytorch-lightning/pull/10134))
+- Deprecated `on_before_accelerator_backend_setup`callback ([#11568](https://github.com/PyTorchLightning/pytorch-lightning/pull/11568))
```

Comment:
This section is for what was deprecated in v1.5. Please move this line up to ~282